pandas/io/tests/test_excel.py

```diff
@@ -58,6 +59,13 @@ def _skip_if_no_xlsxwriter():
         raise nose.SkipTest('xlsxwriter not installed, skipping')
 
 
+def _skip_if_no_ezodf():
```

Comment:
So instead of this, make a new classes, ``ODSReaderTests/ExcelReaderTests``, and a new base class, ``ReadingTests`` where these both inherit. Then almost all of the existing ``ExcelReaderTests`` can be moved to ``ReadingTests``. Use a class variable to hold the suffixes. Add a new function:

```
# for Excel
def _skip_if(self):
    _skip_if_no_xlrd()     

# for ODS
def _skip_if(self):
    _skip_if_no_ezodf()
```

Then remove all of the ``_skip_if_no......`` and replace with ``self._skip_if()``